### PR TITLE
add new event: onRenderedEvent

### PR DESCRIPTION
**Pull request by alanrenmsft**
expose a new event when the grid rendering is completed

### CODE REVIEW

**Comment by alanrenmsft** [comment]
@anthonydresser Hi Anthony, Please take a look at this PR. I will use this event instead of introducing a new event in the angular-slickgrid repo. this one is more useful. Thanks

File: slick.grid.js [hunk 1]
```diff
@@ -2346,6 +2346,7 @@ if (typeof Slick === "undefined") {
       lastRenderedScrollTop = scrollTop;
       lastRenderedScrollLeft = scrollLeft;
       h_render = null;
+      trigger(self.onRendered, { startRow: visible.top, endRow: visible.bottom, grid: self })
```
Comment (anthonydresser) [inline slick.grid.js:2349]: How are you planning on using this?

Comment (alanrenmsft) [reply slick.grid.js:2349]: this is how this event will be consumed. https://github.com/Microsoft/azuredatastudio/pull/2781

Comment (alanrenmsft) [reply slick.grid.js:2349]: i will update this PR: https://github.com/Microsoft/angular2-slickgrid/pull/40 and the PR for ads to use this event.

Comment (anthonydresser) [reply slick.grid.js:2349]: This looks fine, but keep in mind that this will be triggered a lot so any listener should be performant.

**Pull request merged**